src/routes/test/test_units.py

```diff
@@ -0,0 +1,18 @@
+from unittest import TestCase
+from fastapi.testclient import TestClient
+
+def solve_path(path: str):
+    source = 'src.routes.units'
+    return ".".join([source, path])
+
+class GetUnitsRouteTestCase(TestCase):
+    def setUp(self) -> None:
```

Comment:
2 blank spaces after base declarations
```suggestion
    return ".".join([source, path])


class GetUnitsRouteTestCase(TestCase):
    def setUp(self) -> None:
```